pysatNASA/instruments/maven_insitu_kp.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
+"""Module for the MAVEN insitu instruments.
+
+Supports the in situ Key Parameter (kp) data from multiple instruments
```

Comment:
I had thought this kp was like the Kp for the Earth, the multi-hour planetary index related to magnetic field stuff. Unfortunate bit of overlap. 

Double checking on the case for kp, it isn't KP? Also, what is a key parameter for Mars?

Is there a remote kp?

Reply:
Good catch, this should be KP. It is a Key Parameter (team's definition), which includes thermal plasma measurements, etc. Basically the measurements from all instruments that the science team is likely to use. I don't think it includes all parameters. Should be more info in the refs.

Reply:
Double-checking on the update of (kp) to (KP)

Reply:
This should be KP and currently is when I look at the up to date files